common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
+  "message.voicechat.activation_type.ptt": "Push to Talk",
   "message.voicechat.activation_type.voice": "Voice",
-  "message.voicechat.voice_activation": "Voice activation threshold: %s dB",
+  "message.voicechat.voice_activation": "Voice Activation Threshold: %s dB",
   "message.voicechat.voice_activation.disabled": "0 dB deactivates the microphone!",
-  "message.voicechat.adjust_volumes": "Adjust volumes",
+  "message.voicechat.adjust_volumes": "Adjust Volumes",
   "message.voicechat.volume_amplification": "Amplification: %s",
   "message.voicechat.muted": "Muted",
-  "message.voicechat.adjust_volume_player": "Adjust volume of %s",
-  "message.voicechat.no_microphone": "No microphone available",
-  "message.voicechat.no_speaker": "No speaker available",
-  "message.voicechat.select": "Select",
-  "message.voicechat.select_microphone": "Select microphone",
-  "message.voicechat.select_speaker": "Select speaker",
+  "message.voicechat.no_microphone": "No microphones available",
+  "message.voicechat.no_speaker": "No speakers available",
+  "message.voicechat.select_microphone": "Select a Microphone",
+  "message.voicechat.select_speaker": "Select Speakers",
   "message.voicechat.back": "Back",
   "message.voicechat.client_not_connected": "Client not connected",
   "message.voicechat.failed_to_send_ping": "Failed to send ping: %s",
   "message.voicechat.sending_ping": "Sending ping...",
   "message.voicechat.ping_sent_waiting": "Ping sent. Waiting for response...",
-  "message.voicechat.ping_received": "Got a response in %sms",
-  "message.voicechat.ping_received_attempt": "Got a response after %s attempts in %sms",
+  "message.voicechat.ping_received": "Got a response after %s ms",
+  "message.voicechat.ping_received_attempt": "Got a response after %s attempts and %s ms",
   "message.voicechat.ping_retry": "No response. Retrying...",
   "message.voicechat.ping_timed_out": "Request timed out after %s attempts",
   "message.voicechat.icons_hidden": "Voice chat icons hidden",
   "message.voicechat.icons_visible": "Voice chat icons visible",
-  "message.voicechat.incompatible_version": "Your voice chat version is not compatible with the servers version.\nPlease install version %s of %s.",
-  "message.voicechat.incompatible": "Your voice chat version is not compatible with the servers version.",
-  "message.voicechat.group_members": "Group members",
-  "message.voicechat.no_group_members": "This group is empty",
-  "message.voicechat.join_create_group": "Join or Create Group",
-  "message.voicechat.group_name": "Group name",
-  "message.voicechat.leave_group": "Leave group",
+  "message.voicechat.incompatible_version": "Your voice chat client version is not compatible with the server-side version.\nPlease install version %s of %s.",
+  "message.voicechat.incompatible": "Your voice chat client version is not compatible with the server-side version.",
+  "message.voicechat.group_members": "Group members:",
```

Comment:
That's because a colon is the punctuation mark that is most commonly used to introduce a list, and here we have a list of players.

This is what it looks like in the game after the change:
![Group-member-list](https://github.com/user-attachments/assets/de3a147c-0b5f-409e-a094-03da70a554b8)

Reply:
I believe so. Here's what similar tooltips from vanilla Minecraft look like:
![Vanilla-tooltip-equivalents](https://github.com/user-attachments/assets/610211ef-0b3f-4da7-ab52-a095dcb54492)